.github/workflows/create-rancher-pr.yml

```diff
@@ -0,0 +1,129 @@
+name: Create Rancher PR
+
+# This workflow triggers on a push to the main branch. It updates the
+# rancher/shepherd dependency in the rancher/rancher repository and opens a PR.
+on:
+  push:
+    branches:
+      - main
+env:
+  INPUT_SOURCE_URL: ${{ github.event.inputs.source_url }}
+  INPUT_SOURCE_AUTHOR: ${{ github.event.inputs.source_author }}
+
+jobs:
+  create-rancher-pr:
+    permissions:
+      # This is required for the vault action to get an OIDC token
+      id-token: write
+      # This is required to checkout the repository
+      contents: read
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout the current repo (shepherd) to get the commit SHA
+      - name: Checkout shepherd
+        uses: actions/checkout@v4
+
+      - name: Get short SHA
+        id: sha
+        run: echo "short_sha=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
+
+      - name: Retrieve token from vault
+        uses: rancher-eio/read-vault-secrets@main
```

Comment:
Can individual users from the same organization retrieve this, or just the repositories? 


I think, what I am curious about is when you push a commit to your fork, this won't get triggered, right?

Reply:
no it won't get triggered